test/e2e/lib/pages/gutenboarding/designs-page.js

```diff
@@ -11,11 +10,15 @@ export default class DesignLocatorPage extends AsyncBaseContainer {
 
 	async selectFreeDesign() {
 		const freeOptions = await this.driver.findElements( this.freeOptionLocator );
-		await freeOptions[ dataHelper.getRandomInt( 0, freeOptions.length - 1 ) ].click();
-	}
+		if ( freeOptions.length === 0 ) {
+			throw new Error( 'No free designs were found on design selection page' );
+		}
+
+		if ( freeOptions.length === 1 ) {
+			throw new Error( 'There should be more than one design found on the design selection page' );
+		}
```

Comment:
I guess it would be OK to have just one error here? If there should always be more than one design available it should be OK to always display that error only IMO.